python/ray/serve/multiplex.py

```diff
@@ -17,6 +17,7 @@
 from ray.serve.context import _get_global_client, _get_internal_replica_context
 
 logger = logging.getLogger(SERVE_LOGGER_NAME)
+PUSH_MULTIPLEXED_MODEL_IDS_TASK_NAME = "push_multiplexed_model_ids"
```

Comment:
ditto

Reply:
There also isn't a metrics manager in the multiplex wrapper - I'm not familiar with the code here, @sihanwang41 is this something you think we should add?

Reply:
oh, IIUC we can add it into the class for now, since it is pure class level knowledge (instead of module level) for the metric task name. And also suggest to use `_PUSH_MULTIPLEXED_MODEL_IDS_METRIC_TASK_NAME` to make it as internal.

Reply:
Makes sense, I've moved it into the class.